src/main/java/net/atos/zac/app/klanten/model/KlantType.java

```diff
@@ -0,0 +1,17 @@
+package net.atos.zac.app.klanten.model;
+
+public enum KlantType {
+    BURGER,
```

Comment:
Ik hoorde laatst dat de term "Burger" niet meer gehanteerd wordt. Maak er Persoon van.

Reply:
neem ik mee in vervolg ticket